### ISSUE TITLE
Liferay Upgrade 6.2 to version 7

### ISSUE THREAD

**Issue by moyadf** [liferay/liferay-theme-tasks#22]
I have to migrate a theme on liferay 6.2 to Liferay 7; with Nodejs, yeonam and gulp, and when running comand **gulp upgrade** (on windows), the process indicate i having the next error:

{ [Error: EISDIR: illegal operation on a directory, read] errno: -4068, code: 'EISDIR', syscall: 'read' }

```
C:\ThemesLiferay7\portal-osisanitas-theme>gulp upgrade
[11:33:29] Using gulpfile C:\ThemesLiferay7\portal-osisanitas-theme\gulpfile.js
[11:33:29] Starting 'upgrade'...
[11:33:29] Starting 'upgrade:create-backup-files'...
? Would you like to overwrite the existing _backup directory and it's contents?
? Would you like to overwrite the existing _backup directory and it's contents?
 Yes
[11:33:36] Finished 'upgrade:create-backup-files' after 7.33 s
[11:33:36] Starting 'upgrade:black-list'...
[11:33:36] Finished 'upgrade:black-list' after 91 ms
[11:33:36] Starting 'upgrade:replace-compass'...
[11:33:37] Finished 'upgrade:replace-compass' after 276 ms
[11:33:37] Starting 'upgrade:convert-bootstrap'...
{ [Error: EISDIR: illegal operation on a directory, read] errno: -4068, code: 'EISDIR', syscall: 'read' }
```

someone knows what the error is? Thanks...

**Comment by Robert-Frampton** [comment]
Hey @moyadf 

Thanks for reporting this. It looks like it's an issue with the [convert-bootstrap-2-to-3](https://github.com/natecavanaugh/convert-bootstrap-2-to-3) module. I'll go ahead and write up a issue on that project and link it.

**Comment by Robert-Frampton** [comment]
Did this actually stop the gulp task from completing? Or did it just print that error message and continue?

**Comment by Robert-Frampton** [comment]
Fixed with convert-bootstrap-2-to-3 [v1.0.3](https://github.com/natecavanaugh/convert-bootstrap-2-to-3/commit/3054852be7324e0be31c3ccc1f74005f887d9cf3).

If you just reinstall the npm dependencies of your theme everything should work.

**Comment by moyadf** [comment]
Hello @Robert-Frampton 
respect to your question , **whos Did this stop the task from completing Gulp ? Or did it just print that error message and continue ?**, is to start the upgrade gulp, the import of theme is correct, the update to Liferay 7 not.
